language/e2e-tests/src/common_transactions.rs

```diff
@@ -82,6 +82,23 @@ pub fn add_validator_txn(
     )
 }
 
+/// Returns a transaction to update validators' configs and reconfigure
+///   (= emit reconfiguration event and change the epoch)
+pub fn update_validators_and_reconfigure_txn(
```

Comment:
should we just call this "reconfigure"

curious what other folks think about the naming here.

Reply:
I agree - the name is too long... Reconfigure event is also emitted on add_validator_txn and remove_validator_txn.

Reply:
+1 to shorter name